newsrevenuehub/rev-engine
# Dev 1160 associate benefits with rps

#### What's this PR do?
Links Benefits and BenefitLevels with RevenueProgram instead of Org.

#### Why are we doing this? How does it help us?
BenefitLevels should be associated with RevenueProgram and there are no scenarios where Benefits and BenefitLevels associated with Organization.

#### How should this be manually tested?
Navigate to 
/nrhadmin/organizations/benefitlevel/add/ - There will be a dropdown to select RP instead of ORG
/nrhadmin/organizations/benefitlevel/{id}/change/ - Associated RP will be populated instead of ORG
/nrhadmin/organizations/revenueprogram/add/ - Should be able to create a Benefit level
nrhadmin/organizations/revenueprogram/{id}/change/ - Should be able to view the Benefitlevels associated and be able to edit or add new
nrhadmin/organizations/benefit/{}/change/ - Displays associated RP instead of ORG
nrhadmin/organizations/benefit/add/ - Should be able to select RP to associate the benefit with

#### Have automated unit tests been added?
Yes

#### How should this change be communicated to end users?
Unknown

#### Are there any smells or added technical debt to note?
NA

#### Has this been documented? If so, where?
Not yet

#### What are the relevant tickets?
**(Note: Please use syntax [JIRA-123] to link any relevant tickets)**

[DEV-1160]

#### Do any changes need to be made before deployment to production (adding environment variables, for example)?
Should run DB Migration

#### Are there next steps? If so, what? Have tickets been opened for them?
NA
